lib/settings.js

```diff
@@ -157,6 +157,16 @@ async function setReduceMotion (sim, reduceMotion = true) {
   }
 }
 
+async function setReduceTransparency (sim, reduceTransparency = true) {
+  log.debug(`Updating reduce tranceparency. Setting to ${reduceTransparency}.`);
+  const paths = await plistPaths(sim, 'accessibilitySettings');
```

Comment:
oh, these hardcoded identifiers. It needs to be refactored later.

Reply:
I'll take a look at it once this gets merged.